crypto/bio/bio_addr.c

```diff
@@ -951,7 +949,7 @@ int BIO_lookup_ex(const char *host, const char *service, int lookup_type,
             ret = 1;
         }
      err:
-        CRYPTO_THREAD_unlock(bio_lookup_lock);
+            CRYPTO_THREAD_unlock(bio_lookup_lock);
```

Comment:
Please revert this spurious indentation change.

Reply:
Thanks for pointing out, updated.